core/tests/protractor/explorationFeedback.js

```diff
@@ -85,6 +85,8 @@ describe('ExplorationFeedback', function() {
       creatorDashboardPage.getNumberOfFeedbackMessages()
     ).toEqual(1);
     creatorDashboardPage.navigateToExplorationEditor();
+    expect(element(by.css(
```

Comment:
This is too granular for this test. Should be something like explorationEditorPage.isOnFeedbackTab()